Dockerfile

```diff
@@ -1,26 +1,31 @@
-FROM rhel7
+# Using playbook2image as a base
+# See https://github.com/aweiteka/playbook2image for details on the image
+# including documentation for the settings/env vars referenced below
+FROM docker.io/aweiteka/playbook2image:latest
 
-MAINTAINER Troy Dawson <tdawson@redhat.com>
```

Comment:
The original file was moved to `Dockerfile.rhel7`, but because of the name this file is getting compared to the old one.